Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -371,4 +373,17 @@ class FileGenerator {
             p.print("]\n")
         }
     }
+
+    private func generateVersionCheck(printer p: inout CodePrinter) {
+        let v = Version.compatibilityVersion
+        p.print("// If the compiler emits an error on this type, it is because this file\n")
+        p.print("// was generated by a version of the `protoc` Swift plug-in that is\n")
+        p.print("// incompatible with the version of SwiftProtobuf to which you are linking.\n")
+        p.print("// Please ensure that your are building against the same version of the API\n")
+        p.print("// that was used to generate this file.\n")
+        p.print("fileprivate struct _GeneratedWithProtocGenSwiftVersion: SwiftProtobuf.ProtobufAPIVersionCheck {\n")
```

Comment:
Actually, _all_ of the WKTs are currently generated except for Any and Struct/Value/ListValue.

We do not need to drop the `SwiftProtobuf.` module name for code inside the library.